lib/view/item_widget.dart

```diff
@@ -37,20 +40,63 @@ class _ItemWidgetState extends State<ItemWidget> {
           widget.item.isChecked ? TextDecoration.lineThrough : null;
     }
 
-    return ListTile(
-      leading: checkbox,
-      onLongPress: _showOptionsModalSheet,
-      title: Text(widget.item.value,
-          style: itemTextStyle.copyWith(decoration: textDecoration)),
-      onTap: _showEditDialog,
-    );
+    return InkWell(
+        onTap: _showEditDialog,
+        onLongPress: _showOptionsModalSheet,
+        child: Padding(
+            padding:
+                const EdgeInsets.symmetric(horizontal: 16.0, vertical: 8.0),
+            child: Row(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                if (checkbox != null)
+                  Padding(
+                    padding: const EdgeInsets.only(right: 16.0),
+                    child: checkbox,
+                  ),
```

Comment:
If you moved the `Padding` wrapper up to line 34, then this would be much simpler here:
```suggestion
                if (checkbox != null) checkbox
```